setup/landsales/withdraw_from_sales.ts

```diff
@@ -20,7 +20,7 @@ const landSalePrefix = args[0];
   );
 
   for (const sector of sectors) {
-    const landSaleName = `${landSalePrefix}_${sector.sector}`;
+    const landSaleName = `LandPreSale_${sector.sector}`;
```

Comment:
the var ${landSalePrefix} is used line 16 too but you only changed it here

Reply:
Land sale data structure remains the same: `data/landSales/LandPreSale_<deployment number>/*.json`

Only deployments were changed from `LandPreSale_<deployment number>_<sector>.json` to `LandPreSale_<sector>.json`

Now that you mention it, there might be an issue with testnet withdraw, since the name could be hre.network.name || testnet, I'll fix this